package.json

```diff
@@ -41,5 +41,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
is this really a dev dependency? I see it as a clear functional dependency that should be listed as deps and no devDep, am i right?